packages/core-contracts/contracts/utils/SetUtil.sol

```diff
@@ -7,43 +7,43 @@ library SetUtil {
     // ----------------------------------------
 
     struct AddressSet {
-        Bytes32Set raw;
+        mapping(uint => Bytes32Set) raw;
```

Comment:
TBH, I think using a mapping here is overkilling, but I couldn't find a way to allow this only lib to pass the tests without hardcoding it.